src/api/window/window.ts

```diff
@@ -728,7 +727,7 @@ export class _Window extends Base {
      * @tutorial Window.getGroup
      */
     public getGroup(): Promise<Array<_Window>> {
-        return this.wire.sendAction('get-window-group', this.identity).then(({ payload }) => {
+        return this.wire.sendAction<string[]>('get-window-group', this.identity).then(({ payload }) => {
```

Comment:
Again, sendAction returns a list of list of names, not _Window obejcts

Reply:
Review the API for shapes.